### ISSUE TITLE
[Bug]: Module '"@fluentui/react-components"' has no exported member 'Tag'.

### ISSUE THREAD

**Issue by likharsaurav** [microsoft/fluentui#30380]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (8) x64 Intel(R) Core(TM) i7-10610U CPU @ 1.80GHz
    Memory: 6.67 GB / 31.73 GB
  Browsers:
    Edge: Chromium (120.0.2210.133)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

None

### Reproduction

y

## Actual Behavior
There is an error when importing Tag element.
![image](https://github.com/microsoft/fluentui/assets/30959273/0af024c4-c44f-4bf7-8430-8c6ff82463bb)

## Expected Behavior
The Tag element should be imported successfully

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by marcosmoura** [comment]
@likharsaurav What version of `@fluentui/react-components` are you using? Can you provide a minimal Code Sandbox example?

**Comment by likharsaurav** [comment]
This is the version in my package.json - "@fluentui/react-components": "^9.18.2".

For minimal code example, just import the Tag component from above package in VS code editor and it will show error.

**Comment by marcosmoura** [comment]
Tag is not present on the version you are using.
It was released on version [9.34.0](https://github.com/marcosmoura/fluentui/blob/master/packages/react-components/react-components/CHANGELOG.md#9340).
